eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -135,7 +117,6 @@ func (c *ClusterLoader) buildArgs() string {
 		"--provider=eks",
 		"--testconfig=/etc/config/config.yaml",
 		"--run-from-cluster",
-		"--report-dir=/var/reports/cluster-loader",
```

Comment:
Why do we remove this?

Reply:
Most of what I removed is what I've added in the past. This, along with the other removals, is part of the automated CL2 tests, where we don't need it because it adds extra functionality that isn't being used.  We aren't using report-dir because we aren't keeping track of the CL2 test results, just making sure it's being completed.